docs/extras/use_cases/question_answering/index.mdx

```diff
@@ -265,7 +265,7 @@ const stuffChain = loadQAStuffChain(model);
 
 const stuffResult = await stuffChain.call({
   input_documents: relevantDocs,
```

Comment:
More of a question, but is there a reason why this property is in snake_case?

Reply:
`.call` passes those values to the prompt, which has a variable `input_documents`. It's not super clear here - we're working to create an expression language version that will show it more clearly!

Reply:
That makes more sense. Thanks for the clarification ❤️